packages/core/src/database/migrations/z1690911789-create-subgraphMicrocreditBorrowers.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.createTable('subgraph_microcredit_borrowers', {
+            id: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true
+            },
+            userId: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                references: {
+                    model: 'app_user',
+                    key: 'id',
+                },
+                onDelete: 'CASCADE'
+            },
+            lastRepayment: {
+                type: Sequelize.INTEGER,
+                allowNull: true,
+            },
+            lastRepaymentAmount: {
+                type: Sequelize.FLOAT,
+                allowNull: true,
+            },
+            lastDebt: {
+                type: Sequelize.FLOAT,
+                allowNull: true,
+            },
+            amount: {
+                type: Sequelize.FLOAT,
+                allowNull: false,
+            },
+            period: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+            },
+            claimed: {
+                type: Sequelize.INTEGER,
+                allowNull: true, 
+            },
+            createdAt: {
+                type: Sequelize.DATE,
+                allowNull: false
+            },
+            updatedAt: {
+                type: Sequelize.DATE,
+                allowNull: false
+            },
```

Comment:
I would not include timestamps on this table

Reply:
I'm thinking to use the `updatedAt` to know the last updated date and filter the borrowers on subgraph from this date

Reply:
You should always get borrowers from the subgraph that have been updated in the last X minutes, regardless of what `updatedAt` you have on the database.
And you don't need to know when they were last updated. You can just assume they were updated because if the job fails, we will know.
Maybe I'm missing something. At least the `createdAt` it's definitely not needed.